hacspec-scrambledb/scrambledb/src/test_util.rs

```diff
@@ -0,0 +1,48 @@
+use crate::table::{Column, PlainTable};
+
+pub fn generate_plain_table() -> PlainTable {
```

Comment:
Unused? -> Remove?

Reply:
It's used in the unit tests. I would leave it in for now and work on more principled testing in a follow up?